### PR TITLE
[Performance](status code) improve the status code performance

### PR DESCRIPTION
**Pull request by bobhan1**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: be/src/olap/rowset/segment_v2/segment.cpp [hunk 1]
```diff
@@ -367,14 +367,14 @@ Status Segment::lookup_row_key(const Slice& key, bool with_seq_col, RowLocation*
 
     DCHECK(_pk_index_reader != nullptr);
     if (!_pk_index_reader->check_present(key_without_seq)) {
-        return Status::NotFound("Can't find key in the segment");
+        return Status::NotFound("");
```
Comment (yiguolei) [inline be/src/olap/rowset/segment_v2/segment.cpp:370]: Is this pr will really improve performance???

**Comment by yiguolei** [comment]
run buildall

**Comment by platoneko** [comment]
Such modification seems meaningless.

**Review by platoneko — COMMENTED** [review]
Seems meaningless

**Comment by bobhan1** [comment]
> Seems meaningless

I misuse the ASAN build to do profiling and get the wrong result

**Pull request closed without merging**